sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/CalculateSchemas.java

```diff
@@ -0,0 +1,93 @@
+package org.apache.beam.sdk.io.gcp.bigquery;
+
+/** Created by relax on 4/30/17. */
```

Comment:
Doesn't look done to me: I'm looking at the full diff for this PR, and it's still broken.